tests/ctst/features/quotas/Quotas.feature

```diff
@@ -0,0 +1,96 @@
+Feature: Quota Management for APIs
+    This feature ensures that quotas are correctly set and honored
+    for different APIs.
+
+    @2.6.0
+    @PreMerge
+    @Quotas
+    @CronJob
+    @DataWrite
+    Scenario Outline: Quotas are evaluated during write operations
+        Given an action "<action>"
```

Comment:
We usually don't change indentation after `Scenario Outline` either we do it everywhere, either nowhere.. What do you think ?

Reply:
We change the indentation everywhere, but sometimes it's only 2 spaces
All bucket policies, IAM policies, cloudserverAuth follow this (standard) format